src/main/java/org/folio/innreach/batch/contribution/service/ContributionJobRunner.java

```diff
@@ -178,47 +180,51 @@ public void cancelContributionIfRetryExhausted(UUID centralServerId) {
     contributionService.cancelCurrent(centralServerId);
   }
 
-  public void runInstanceContribution(UUID centralServerId, Instance instance) {
-    log.info("Ongoing: validating instance {} for contribution to central server {}", instance.getId(), centralServerId);
-
-    boolean eligibleInstance = isEligibleForContribution(centralServerId, instance);
-    boolean contributedInstance = isContributed(centralServerId, instance);
-    log.info("Ongoing: eligibleInstance: {}, contributedInstance: {}", eligibleInstance, contributedInstance);
-    if (!eligibleInstance && !contributedInstance) {
-      log.info("Ongoing: skipping ineligible and non-contributed instance");
-      return;
-    }
-
-    runOngoing(centralServerId, (ctx, statistics) -> {
-      log.info("Ongoing: starting ongoing instance contribution job {}, centralServerId: {}", ctx, centralServerId);
-
+  public void runOngoingInstanceContribution(UUID centralServerId, Instance instance, OngoingContributionStatus ongoingContributionStatus) {
+    try {
+      log.info("runOngoingInstanceContribution:: validating instance {} for contribution to central server {}", instance.getId(), centralServerId);
+      boolean eligibleInstance = isEligibleForContribution(centralServerId, instance);
+      boolean contributedInstance = isContributed(centralServerId, instance);
+      log.info("runOngoingInstanceContribution:: eligibleInstance: {}, contributedInstance: {}", eligibleInstance, contributedInstance);
+      if (!eligibleInstance && !contributedInstance) {
+        log.info("runOngoingInstanceContribution:: skipping ineligible and non-contributed instance");
```

Comment:
Can we log the instance id here? Because having a message about skipping without knowing which one was skipped probably isn't going to help people looking at the logs to troubleshoot a production issue.

Reply:
I added it